<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" ignored="0" total="4" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="21009" started-at="2020-10-05T00:03:22 NZDT" finished-at="2020-10-05T00:03:43 NZDT">
    <groups>
    </groups>
    <test name="Default test" duration-ms="21009" started-at="2020-10-05T00:03:22 NZDT" finished-at="2020-10-05T00:03:43 NZDT">
      <class name="com.qa.testcase.KiwiSaverCalctest">
        <test-method status="PASS" signature="Setup()[pri:0, instance:com.qa.testcase.KiwiSaverCalctest@6caf0677]" name="Setup" is-config="true" duration-ms="4607" started-at="2020-10-05T00:03:23 NZDT" finished-at="2020-10-05T00:03:27 NZDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="FAIL" signature="TC1_ShowInfoMessageForAge()[pri:0, instance:com.qa.testcase.KiwiSaverCalctest@6caf0677]" name="TC1_ShowInfoMessageForAge" duration-ms="7" started-at="2020-10-05T00:03:27 NZDT" finished-at="2020-10-05T00:03:27 NZDT">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.qa.testcase.KiwiSaverCalctest.TC1_ShowInfoMessageForAge(KiwiSaverCalctest.java:39)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC1_ShowInfoMessageForAge -->
        <test-method status="PASS" signature="CloseApplication()[pri:0, instance:com.qa.testcase.KiwiSaverCalctest@6caf0677]" name="CloseApplication" is-config="true" duration-ms="636" started-at="2020-10-05T00:03:27 NZDT" finished-at="2020-10-05T00:03:28 NZDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseApplication -->
        <test-method status="PASS" signature="Setup()[pri:0, instance:com.qa.testcase.KiwiSaverCalctest@6caf0677]" name="Setup" is-config="true" duration-ms="4544" started-at="2020-10-05T00:03:28 NZDT" finished-at="2020-10-05T00:03:32 NZDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="FAIL" signature="VerifyUS1TC1(java.lang.String, java.lang.String)[pri:0, instance:com.qa.testcase.KiwiSaverCalctest@6caf0677]" name="VerifyUS1TC1" duration-ms="3" started-at="2020-10-05T00:03:32 NZDT" data-provider="data1" finished-at="2020-10-05T00:03:32 NZDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[30]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[82000]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.qa.testcase.KiwiSaverCalctest.VerifyUS1TC1(KiwiSaverCalctest.java:59)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyUS1TC1 -->
        <test-method status="PASS" signature="CloseApplication()[pri:0, instance:com.qa.testcase.KiwiSaverCalctest@6caf0677]" name="CloseApplication" is-config="true" duration-ms="671" started-at="2020-10-05T00:03:32 NZDT" finished-at="2020-10-05T00:03:33 NZDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseApplication -->
        <test-method status="PASS" signature="Setup()[pri:0, instance:com.qa.testcase.KiwiSaverCalctest@6caf0677]" name="Setup" is-config="true" duration-ms="4584" started-at="2020-10-05T00:03:33 NZDT" finished-at="2020-10-05T00:03:38 NZDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="FAIL" signature="VerifyUS2TC2(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.qa.testcase.KiwiSaverCalctest@6caf0677]" name="VerifyUS2TC2" duration-ms="2" started-at="2020-10-05T00:03:38 NZDT" data-provider="data2" finished-at="2020-10-05T00:03:38 NZDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[45]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[100000]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[90]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[290000]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.qa.testcase.KiwiSaverCalctest.VerifyUS2TC2(KiwiSaverCalctest.java:78)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyUS2TC2 -->
        <test-method status="PASS" signature="CloseApplication()[pri:0, instance:com.qa.testcase.KiwiSaverCalctest@6caf0677]" name="CloseApplication" is-config="true" duration-ms="670" started-at="2020-10-05T00:03:38 NZDT" finished-at="2020-10-05T00:03:38 NZDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseApplication -->
        <test-method status="PASS" signature="Setup()[pri:0, instance:com.qa.testcase.KiwiSaverCalctest@6caf0677]" name="Setup" is-config="true" duration-ms="4569" started-at="2020-10-05T00:03:38 NZDT" finished-at="2020-10-05T00:03:43 NZDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="FAIL" signature="VerifyUS2TC3(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.qa.testcase.KiwiSaverCalctest@6caf0677]" name="VerifyUS2TC3" duration-ms="1" started-at="2020-10-05T00:03:43 NZDT" data-provider="data3" finished-at="2020-10-05T00:03:43 NZDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[55]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1400000]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200000]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.qa.testcase.KiwiSaverCalctest.VerifyUS2TC3(KiwiSaverCalctest.java:97)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyUS2TC3 -->
        <test-method status="PASS" signature="CloseApplication()[pri:0, instance:com.qa.testcase.KiwiSaverCalctest@6caf0677]" name="CloseApplication" is-config="true" duration-ms="639" started-at="2020-10-05T00:03:43 NZDT" finished-at="2020-10-05T00:03:43 NZDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseApplication -->
      </class> <!-- com.qa.testcase.KiwiSaverCalctest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
